make/Main.gmk

```diff
@@ -251,6 +252,14 @@ endef
 
 $(foreach v, $(JVM_VARIANTS), $(eval $(call DeclareHotspotLibsRecipe,$v)))
 
+define DeclareHotspotStaticLibsRecipe
+  hotspot-$1-static-libs:
+	+($(CD) $(TOPDIR)/make/hotspot && $(MAKE) $(MAKE_ARGS) -f lib/CompileLibraries.gmk \
+	    JVM_VARIANT=$1 STATIC_LIBS=true)
+endef
+
+$(foreach v, $(JVM_VARIANTS), $(eval $(call DeclareHotspotStaticLibsRecipe,$v)))
```

Comment:
If we are only using the libjvm.a from the main variant, we don't need to generate top level rules for all variants, or we could at least limit the dependency declaration at line 1109 to only need the main variant.

Reply:
Changed to define/use the `libjvm.a` rule for the main variant only as suggested, thanks.